epochStart/bootstrap/storageProcess.go

```diff
@@ -74,24 +81,26 @@ func NewStorageEpochStartBootstrap(args ArgsStorageEpochStartBootstrap) (*storag
 	return sesb, nil
 }
 
-func createEpochStartBootstrapper(args ArgsStorageEpochStartBootstrap) (*epochStartBootstrap, error) {
-	esb, err := NewEpochStartBootstrap(args.ArgsEpochStartBootstrap)
+func createEpochStartBootstrapper(args ArgsStorageEpochStartBootstrap) (EpochStartBootstrapper, error) {
+	fact, err := NewEpochStartBootstrapperFactory()
 	if err != nil {
 		return nil, err
 	}
 
 	switch args.ChainRunType {
 	case common.ChainRunTypeRegular:
-		return esb, nil
+		args.EpochStartBootstrapperFactoryHandler = fact
```

Comment:
function that changes (locally, but changes) the arguments. Please refactor,

Reply:
the switch will be removed in a next PR - added \\TODO  to remove switch

Reply:
great